### PR TITLE
feat: sentry integration

### PR DESCRIPTION
**Pull request by thesephirot**
introducing 2 config and 1 secret variables for sentry

### CODE REVIEW

**Comment by fforootd** [comment]
IMO we should also add the release tag.

It makes it easier to check against a specific version.

**Comment by fforootd** [comment]
FYI, instrumentation is available here https://docs.sentry.io/platforms/go/ and also here https://docs.sentry.io/platforms/javascript/guides/angular/

It could be interesting to rollout the angular client as well

**Comment by thesephirot** [comment]
> IMO we should also add the release tag.
> 
> It makes it easier to check against a specific version.

Fully agree, I am currently on it

File: cmd/zitadel/main.go [hunk 1]
```diff
@@ -89,6 +92,10 @@ const (
 )
 
 func main() {
+	err := sentry.Init(sentry.ClientOptions{})
+	if err != nil {
+		log.Fatalf("sentry.Init: %s", err)
+	}
```
Comment (livio-a) [inline cmd/zitadel/main.go:98]: IMHO we should only start sentry by setting a flag or at least have a flag to disable it

**Review by livio-a — CHANGES_REQUESTED** [review]

File: cmd/zitadel/main.go (hunk 1 above)
Comment (fforootd) [reply cmd/zitadel/main.go:98]: Should we not just rely on the input values?
E.g. when no config is given we don't load this?

Comment (thesephirot) [reply cmd/zitadel/main.go:98]: sound reasonable, a shell variable should fix that

Comment (fforootd) [reply cmd/zitadel/main.go:98]: why exactly a shell variable?

Comment (thesephirot) [reply cmd/zitadel/main.go:98]: thats what ops people only care about :D
call it "variable" 

on it

**Review by adlerhurst — APPROVED** [review]

**Review by livio-a — APPROVED** [review]

**Pull request merged**